lib/middleware.js

```diff
@@ -496,6 +497,18 @@ var middlewares = module.exports = {
     });
     res.end();
   },
+  imageSave: function (req, res) {
+    var stringStream = new StringStream('TARTARTARTARTAR');
+    res.setHeader('Content-Type', 'application/x-tar');
+    res.status(200);
+    stringStream.pipe(res);
+  },
+  imageLoad: function (req, res) {
+    req.on('data', noop);
+    req.on('end', function () {
+      res.sendStatus(200);
+    });
+  },
```

Comment:
could we not have save/load actually use the images to send _something_ that is accurate and useful? Additionally, the `findImage` middleware should be run prior to `imageSave` to validate the image actually exists in the mock (not _always_ say that the image exists)